gazebo/physics/Base.cc

```diff
@@ -383,15 +384,19 @@ common::URI Base::URI() const
   {
     if (p->GetParent())
     {
-      uri.Path().PushFront(p->GetName());
+      std::string escapedParentName = p->GetName();
+      boost::replace_all(escapedParentName, "/", "%2f");
```

Comment:
use `common::replaceAll` instead of `boost`? I do still see `boost` being used in gazebo so maybe not a big deal

Reply:
I switched to `common::replaceAll` in https://github.com/osrf/gazebo/pull/3187/commits/7edb013c5c86738ea6489609f042b099f9707c67